sdk/metricsadvisor/ai-metrics-advisor/src/models.ts

```diff
@@ -927,6 +949,14 @@ export interface DataPointAnomaly {
    * only return for alerting anomaly result
    */
   status?: AnomalyStatus;
+  /**
+   * value of the anomaly
+   */
+  value?: number;
```

Comment:
should `value` and `expectedValue` be readonly?  Do they always have values (non-optional)?

Reply:
On the generated layer these are optional and readonly. Anu has asked on the service channel to clarify if they can have null values. Maybe we can wait till then.